### PR TITLE
Add DVCLive-Quickstart

### PR DESCRIPTION
**Pull request by daavoo**
To be used in Colab

Test in:

https://colab.research.google.com/github/iterative/dvclive/blob/colab/examples/DVCLive-Quickstart.ipynb

### CODE REVIEW

File: examples/DVCLive-Quickstart.ipynb [hunk 1]
```diff
@@ -0,0 +1,354 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-Quickstart.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (daavoo) [inline examples/DVCLive-Quickstart.ipynb:8]: This link would work once we merge on the main branch, in the meantime use  https://colab.research.google.com/github/iterative/dvclive/blob/colab/examples/DVCLive-Quickstart.ipynb

File: examples/DVCLive-Quickstart.ipynb [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# You can modify these parameters to see how they affect the training\n",
+    "# And run the cell several times\n",
```
Comment (daavoo) [inline examples/DVCLive-Quickstart.ipynb:214]: Maybe we should include a `for` loop running multiple experiments by default

**Comment by dberenbaum** [comment]
Do you think we could use example-get-started-experiments for this?

**Comment by shcheklein** [comment]
I love the idea and direction and how fast you are improving it, @daavoo . I'm even fine to merge this :)

> Do you think we could use example-get-started-experiments for this?

But I agree on this also. We should try to use the same project. It reduces the surface of things to maintain and remember for us.

**Comment by dberenbaum** [comment]
I think having a colab notebook for example-get-started-experiments would be nice to have anyway.

**Comment by daavoo** [comment]
> I think having a colab notebook for example-get-started-experiments would be nice to have anyway.
> 
> Edit: agreed it's a minor note though and this looks good in general to me.

I did one for the blog post release: 

https://colab.research.google.com/drive/1NTivljRYiySMJn-SHeWQSycBmSOVUbvA

What is hard is to make it "based on"/"linked to" the one from the GitHub repo, as this quickstart, because workflows are just too different in Colab. The notebook would have to be filled with conditions to alter the workflow depending on whether you are executing it in Colab or in a different context like a codespace or local machine.

This quickstart works fine because we don't really expect to also use it in a real git-based workflow. It is a one-time visit where you discard the end results. 

To make it work git-based, you need to run `git clone` in colab, `cd` to the repo, somehow dump the current notebook changes to a file and track it with git, etc. It just becomes nonsense.

**Comment by dberenbaum** [comment]
> I did one for the blog post release:
> 
> https://colab.research.google.com/drive/1NTivljRYiySMJn-SHeWQSycBmSOVUbvA

LGTM. My suggestion was about reusing the use case, not tracking the colab notebook in the repo, and I think that notebook already does a good enough job of it. Can we combine these two notebooks so you use the experiment-get-started use case but highlight the DVCLive stuff and hide the irrelevant stuff?

**Comment by dberenbaum** [comment]
We could optionally add it to example-repos-dev and/or example-get-started-experiments to keep changes in sync, but it the notebook itself doesn't need to "depend" on the repo.

**Comment by daavoo** [comment]
> We could optionally add it to example-repos-dev and/or example-get-started-experiments to keep changes in sync, but it the notebook itself doesn't need to "depend" on the repo.

I was actually thinking on adding it to `dvclive/examples`, and making that folder a little about `Colab` intro, as `example-repos-dev` is more about reproducing and generating a full git repo.

**Comment by daavoo** [comment]
Merging this one, lets discuss improvements and ideas as follow ups

**Pull request merged**